webpack.config.js

```diff
@@ -23,6 +23,12 @@ const config = {
           presets: ['@babel/preset-env', '@babel/preset-react'],
           plugins: ['@babel/plugin-transform-runtime']
         }
+     },
+     {
+       test: /\.m?js/,
```

Comment:
Sorry, but where / why are we using .mjs files?

Reply:
There was a problem using linked bundles.. Removing!